### PR TITLE
Make v5 -> v6 migration verbose

### PR DESCRIPTION
**Pull request by DL6ER**
# What does this implement/fix?

Make v5 -> v6 migration verbose by default (not only when `debug.config = true`). This will happen only once and may greatly ease debugging in case unintended side-effects (most often it will by typos in the old `setupVars.conf`).

---

**Related issue or feature (if applicable):** N/A

**Pull request in [docs](https://github.com/pi-hole/docs) with documentation (if applicable):** N/A

---
**By submitting this pull request, I confirm the following:** 

1. I have read and understood the [contributors guide](https://docs.pi-hole.net/guides/github/contributing/), as well as this entire template. I understand which branch to base my commits and Pull Requests against. 
2. I have commented my proposed changes within the code.
3. I am willing to help maintain this change if there are issues with it later.
4. It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
5. I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

## Checklist:

- [x] The code change is tested and works locally.
- [x] I based my code and PRs against the repositories `developmental` branch.
- [x] I [signed off](https://docs.pi-hole.net/guides/github/how-to-signoff/) all commits. Pi-hole enforces the [DCO](https://docs.pi-hole.net/guides/github/dco/) for all contributions
- [x] I [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) all my commits. Pi-hole requires signatures to verify authorship
- [x] I have read the above and my PR is ready for review.

### CODE REVIEW

File: src/api/history.c [hunk 1]
```diff
@@ -18,8 +18,6 @@
 #include "overTime.h"
 // config struct
 #include "config/config.h"
-// read_setupVarsconf()
-#include "config/setupVars.h"
```
Comment (DL6ER) [inline src/api/history.c:22]: Unnecessary here, found when `grep`ing for `read_setupVarsconf()` to see where it is used

**Comment by PromoFaux** [comment]
A couple of observations:

Blank Line between upstreams
```
pihole  | 2024-08-26 20:47:07.374 BST [51M] INFO: setupVars.conf:WEBTHEME -> Not set
pihole  | 2024-08-26 20:47:07.374 BST [51M] INFO: setupVars.conf:PIHOLE_DNS_1 -> Setting dns.upstreams[1] = 8.8.8.8
pihole  | 
pihole  | 2024-08-26 20:47:07.374 BST [51M] INFO: setupVars.conf:PIHOLE_DNS_2 -> Setting dns.upstreams[2] = 8.8.4.4
pihole  | 
pihole  | 2024-08-26 20:47:07.374 BST [51M] INFO: setupVars.conf:PIHOLE_DOMAIN -> Not set
```

I think I spotted this when testing on `development-v6` yesterday, but is this expected?

```
pihole  | 2024-08-26 20:47:07.373 BST [51M] INFO: Config file /etc/pihole/pihole.toml not available (r): No such file or directory
pihole  | 2024-08-26 20:47:07.373 BST [51M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.1 not available (r): No such file or directory
pihole  | 2024-08-26 20:47:07.373 BST [51M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.2 not available (r): No such file or directory
pihole  | 2024-08-26 20:47:07.373 BST [51M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.3 not available (r): No such file or directory
pihole  | 2024-08-26 20:47:07.373 BST [51M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.4 not available (r): No such file or directory
pihole  | 2024-08-26 20:47:07.373 BST [51M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.5 not available (r): No such file or directory
pihole  | 2024-08-26 20:47:07.373 BST [51M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.6 not available (r): No such file or directory
pihole  | 2024-08-26 20:47:07.373 BST [51M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.7 not available (r): No such file or directory
pihole  | 2024-08-26 20:47:07.373 BST [51M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.8 not available (r): No such file or directory
pihole  | 2024-08-26 20:47:07.373 BST [51M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.9 not available (r): No such file or directory
pihole  | 2024-08-26 20:47:07.373 BST [51M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.10 not available (r): No such file or directory
pihole  | 2024-08-26 20:47:07.373 BST [51M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.11 not available (r): No such file or directory
pihole  | 2024-08-26 20:47:07.373 BST [51M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.12 not available (r): No such file or directory
pihole  | 2024-08-26 20:47:07.373 BST [51M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.13 not available (r): No such file or directory
pihole  | 2024-08-26 20:47:07.373 BST [51M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.14 not available (r): No such file or directory
```

File: src/config/setupVars.c [hunk 2]
```diff
@@ -295,13 +316,12 @@ static void get_conf_upstream_servers_from_setupVars(struct conf_item *conf_item
...
 		if(value != NULL)
 		{
-			log_debug(DEBUG_CONFIG, "%s = %s\n", server_key, value);
 			// Add string to our JSON array
 			cJSON *item = cJSON_CreateString(value);
 			cJSON_AddItemToArray(conf_item->v.json, item);
 
-			log_debug(DEBUG_CONFIG, "setupVars.conf:PIHOLE_DNS_%u -> Setting %s[%u] = %s\n",
-			          j, conf_item->k, j, item->valuestring);
+			log_info("setupVars.conf:PIHOLE_DNS_%u -> Setting %s[%u] = %s\n",
```
[the remark this answers is not in the document]
Comment (PromoFaux) [reply src/config/setupVars.c:323]: Yes, that.

```
pihole  | 2024-08-26 20:55:48.933 BST [51M] INFO: setupVars.conf:WEBTHEME -> Not set
pihole  | 2024-08-26 20:55:48.933 BST [51M] INFO: setupVars.conf:PIHOLE_DNS_1 -> Setting dns.upstreams[1] = 8.8.8.8
pihole  | 2024-08-26 20:55:48.933 BST [51M] INFO: setupVars.conf:PIHOLE_DNS_2 -> Setting dns.upstreams[2] = 8.8.4.4
pihole  | 2024-08-26 20:55:48.933 BST [51M] INFO: setupVars.conf:PIHOLE_DOMAIN -> Not set
```

**Review by yubiuser — COMMENTED** [review]
I did a frehs v5 install in a VM, run 
```
echo "development-v6" | sudo tee /etc/pihole/ftlbranch 
pihole checkout core development-v6
```
And this is the output. There is one error and some warnings. Except of this, the output looks fine.

```
[2024-09-02 04:09:30.793 13398M] INFO: FTL is running as user pihole (UID 995)
[2024-09-02 04:09:30.793 13398M] Reloading DNS cache
[2024-09-02 04:09:30.794 13398/T13460] gravityDB_count(SELECT value FROM info WHERE property = 'gravity_count';) - SQL error step no more rows available
[2024-09-02 04:09:30.794 13398/T13460] Count of gravity domains not available. Please run pihole -g
[2024-09-02 04:09:30.799 13398/T13460] Compiled 0 whitelist and 0 blacklist regex filters for 0 clients in 5.0 msec
[2024-09-02 04:09:30.799 13398/T13460] Blocking status is enabled
[2024-09-02 04:09:31.768 13398M] Reloading DNS cache
[2024-09-02 04:09:31.820 13398/T13460] SQLite3 message: file renamed while open: /etc/pihole/gravity.db (28)
[2024-09-02 04:09:31.820 13398/T13460] Compiled 0 whitelist and 0 blacklist regex filters for 0 clients in 0.1 msec
[2024-09-02 04:09:31.821 13398/T13460] Blocking status is enabled
[2024-09-02 04:09:31.888 13398M] Received: Real-time signal 0 (34 -> 0)
[2024-09-02 04:09:31.921 13398/T13460] Compiled 0 whitelist and 0 blacklist regex filters for 0 clients in 0.1 msec
[2024-09-02 04:09:31.921 13398/T13460] Blocking status is enabled
2024-09-02 04:11:25.652 EDT [14103M] INFO: Config file /etc/pihole/pihole.toml not available (r): No such file or directory
2024-09-02 04:11:25.661 EDT [14112M] INFO: Config file /etc/pihole/pihole.toml not available (r): No such file or directory
2024-09-02 04:11:25.661 EDT [14113M] INFO: Config file /etc/pihole/pihole.toml not available (r): No such file or directory
2024-09-02 04:11:25.667 EDT [14115M] INFO: Config file /etc/pihole/pihole.toml not available (r): No such file or directory
2024-09-02 04:11:25.670 EDT [14116M] INFO: Config file /etc/pihole/pihole.toml not available (r): No such file or directory
2024-09-02 04:11:25.679 EDT [14117M] INFO: Config file /etc/pihole/pihole.toml not available (r): No such file or directory
2024-09-02 04:11:25.687 EDT [14118M] INFO: Config file /etc/pihole/pihole.toml not available (r): No such file or directory
2024-09-02 04:11:25.701 EDT [14119M] INFO: Config file /etc/pihole/pihole.toml not available (r): No such file or directory
2024-09-02 04:11:25.954 EDT [14138M] INFO: Config file /etc/pihole/pihole.toml not available (r): No such file or directory
2024-09-02 04:11:25.973 EDT [14152M] INFO: Config file /etc/pihole/pihole.toml not available (r): No such file or directory
2024-09-02 04:11:25.973 EDT [14152M] INFO: ########## FTL started on osboxes! ##########
2024-09-02 04:11:25.973 EDT [14152M] INFO: FTL branch: tweak/verbose_migration
2024-09-02 04:11:25.973 EDT [14152M] INFO: FTL version: vDev-453cdef
2024-09-02 04:11:25.973 EDT [14152M] INFO: FTL commit: 453cdefb
2024-09-02 04:11:25.973 EDT [14152M] INFO: FTL date: 2024-08-26 21:59:41 +0200
2024-09-02 04:11:25.973 EDT [14152M] INFO: FTL user: pihole
2024-09-02 04:11:25.973 EDT [14152M] INFO: Compiled for linux/amd64 (compiled on CI) using cc (Alpine 13.2.1_git20240309) 13.2.1 20240309
2024-09-02 04:11:25.973 EDT [14152M] INFO: Config file /etc/pihole/pihole.toml not available (r): No such file or directory
2024-09-02 04:11:25.973 EDT [14152M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.1 not available (r): No such file or directory
2024-09-02 04:11:25.973 EDT [14152M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.2 not available (r): No such file or directory
2024-09-02 04:11:25.973 EDT [14152M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.3 not available (r): No such file or directory
2024-09-02 04:11:25.974 EDT [14152M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.4 not available (r): No such file or directory
2024-09-02 04:11:25.974 EDT [14152M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.5 not available (r): No such file or directory
2024-09-02 04:11:25.974 EDT [14152M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.6 not available (r): No such file or directory
2024-09-02 04:11:25.974 EDT [14152M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.7 not available (r): No such file or directory
2024-09-02 04:11:25.974 EDT [14152M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.8 not available (r): No such file or directory
2024-09-02 04:11:25.974 EDT [14152M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.9 not available (r): No such file or directory
2024-09-02 04:11:25.974 EDT [14152M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.10 not available (r): No such file or directory
2024-09-02 04:11:25.974 EDT [14152M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.11 not available (r): No such file or directory
2024-09-02 04:11:25.974 EDT [14152M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.12 not available (r): No such file or directory
2024-09-02 04:11:25.974 EDT [14152M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.13 not available (r): No such file or directory
2024-09-02 04:11:25.974 EDT [14152M] INFO: Config backup file /etc/pihole/config_backups/pihole.toml.14 not available (r): No such file or directory
2024-09-02 04:11:25.974 EDT [14152M] INFO: No config file nor backup available, using defaults
2024-09-02 04:11:25.974 EDT [14152M] INFO: Reading legacy config files from /etc/pihole/pihole-FTL.conf
2024-09-02 04:11:25.974 EDT [14152M] INFO:    PIDFILE: Empty path is not possible, using default
2024-09-02 04:11:25.974 EDT [14152M] INFO:    SETUPVARSFILE: Empty path is not possible, using default
2024-09-02 04:11:25.974 EDT [14152M] INFO:    MACVENDORDB: Empty path is not possible, using default
2024-09-02 04:11:25.974 EDT [14152M] INFO:    GRAVITYDB: Empty path is not possible, using default
2024-09-02 04:11:25.974 EDT [14152M] INFO:    WEBROOT: Empty path is not possible, using default
2024-09-02 04:11:25.974 EDT [14152M] INFO:    WEBHOME: Empty path is not possible, using default
2024-09-02 04:11:25.974 EDT [14152M] INFO:    API_INFO_LOG: Empty path is not possible, using default
2024-09-02 04:11:25.974 EDT [14152M] INFO:    WEBDOMAIN: Empty path is not possible, using default
2024-09-02 04:11:25.974 EDT [14152M] INFO: Moving /etc/pihole/pihole-FTL.conf to /etc/pihole/pihole-FTL.conf.bck
2024-09-02 04:11:25.974 EDT [14152M] INFO: Migrating config from /etc/pihole/setupVars.conf
2024-09-02 04:11:25.974 EDT [14152M] INFO: setupVars.conf:WEBPASSWORD -> Setting webserver.api.pwhash to 5f8402b2cb530201f615f871ef70c30a1327faad65803ab331b2fe59398474ae
2024-09-02 04:11:25.974 EDT [14152M] INFO: setupVars.conf:BLOCKING_ENABLED -> Setting dns.blocking.active to true
2024-09-02 04:11:25.974 EDT [14152M] INFO: setupVars.conf:TEMPERATURE_LIMIT -> Not set
2024-09-02 04:11:25.974 EDT [14152M] INFO: setupVars.conf:TEMPERATURE_UNIT -> Not set
2024-09-02 04:11:25.974 EDT [14152M] INFO: setupVars.conf:WEBUIBOXEDLAYOUT -> Not set
2024-09-02 04:11:25.974 EDT [14152M] INFO: setupVars.conf:WEBTHEME -> Not set
2024-09-02 04:11:25.974 EDT [14152M] INFO: setupVars.conf:PIHOLE_DNS_1 -> Setting dns.upstreams[1] = 8.8.8.8
2024-09-02 04:11:25.974 EDT [14152M] INFO: setupVars.conf:PIHOLE_DNS_2 -> Setting dns.upstreams[2] = 8.8.4.4
2024-09-02 04:11:25.975 EDT [14152M] INFO: setupVars.conf:PIHOLE_DOMAIN -> Not set
2024-09-02 04:11:25.975 EDT [14152M] INFO: setupVars.conf:DNS_FQDN_REQUIRED -> Setting dns.domainNeeded to true
2024-09-02 04:11:25.976 EDT [14152M] INFO: setupVars.conf:DNS_FQDN_REQUIRED -> Setting dns.expandHosts to true
2024-09-02 04:11:25.976 EDT [14152M] INFO: setupVars.conf:DNS_bogusPriv -> Not set
2024-09-02 04:11:25.976 EDT [14152M] INFO: setupVars.conf:DNSSEC -> Not set
2024-09-02 04:11:25.976 EDT [14152M] INFO: setupVars.conf:PIHOLE_INTERFACE -> Setting dns.interface to enp0s3
2024-09-02 04:11:25.976 EDT [14152M] INFO: setupVars.conf:HOSTRECORD -> Not set
2024-09-02 04:11:25.976 EDT [14152M] INFO: setupVars.conf:DNSMASQ_LISTENING -> Setting dns.listeningMode to LOCAL
2024-09-02 04:11:25.976 EDT [14152M] INFO: setupVars.conf:REV_SERVER -> Not set
2024-09-02 04:11:25.976 EDT [14152M] INFO: setupVars.conf:DHCP_ACTIVE -> Not set
2024-09-02 04:11:25.976 EDT [14152M] INFO: setupVars.conf:DHCP_START -> Not set
2024-09-02 04:11:25.976 EDT [14152M] INFO: setupVars.conf:DHCP_END -> Not set
2024-09-02 04:11:25.976 EDT [14152M] INFO: setupVars.conf:DHCP_ROUTER -> Not set
2024-09-02 04:11:25.976 EDT [14152M] INFO: setupVars.conf:DHCP_LEASETIME -> Not set
2024-09-02 04:11:25.976 EDT [14152M] INFO: setupVars.conf:DHCP_IPv6 -> Not set
2024-09-02 04:11:25.976 EDT [14152M] INFO: setupVars.conf:DHCP_RAPID_COMMIT -> Not set
2024-09-02 04:11:25.976 EDT [14152M] INFO: setupVars.conf:queryLogging -> Not set
2024-09-02 04:11:25.976 EDT [14152M] INFO: setupVars.conf:GRAVITY_TMPDIR -> Not set
2024-09-02 04:11:25.976 EDT [14152M] INFO: setupVars.conf:WEB_PORTS -> Not set
2024-09-02 04:11:25.976 EDT [14152M] INFO: Moved /etc/pihole/setupVars.conf to /etc/pihole/setupVars.conf.old
2024-09-02 04:11:25.976 EDT [14152M] INFO: Migration complete
2024-09-02 04:11:25.977 EDT [14152M] INFO: Moving /etc/pihole/custom.list to /etc/pihole/custom.list.bck
2024-09-02 04:11:25.977 EDT [14152M] INFO: Config initialized with webserver ports 8080 (HTTP) and 443 (HTTPS), IPv6 support is enabled
2024-09-02 04:11:25.978 EDT [14152M] INFO: Wrote config file:
2024-09-02 04:11:25.978 EDT [14152M] INFO:  - 148 total entries
2024-09-02 04:11:25.978 EDT [14152M] INFO:  - 142 entries are default
2024-09-02 04:11:25.978 EDT [14152M] INFO:  - 6 entries are modified
2024-09-02 04:11:25.978 EDT [14152M] INFO:  - 0 entries are forced through environment
2024-09-02 04:11:25.978 EDT [14152M] INFO: Config file written to /etc/pihole/pihole.toml
2024-09-02 04:11:25.984 EDT [14152M] CRIT: pihole-FTL is already running!
2024-09-02 04:11:25.984 EDT [14152M] INFO: bash (PID 14155) ──> pihole-FTL (PID 14162, started 2024-09-02 04:11:25 EDT)
2024-09-02 04:11:31.145 EDT [14359M] INFO: ########## FTL started on osboxes! ##########
2024-09-02 04:11:31.146 EDT [14359M] INFO: FTL branch: tweak/verbose_migration
2024-09-02 04:11:31.146 EDT [14359M] INFO: FTL version: vDev-453cdef
2024-09-02 04:11:31.146 EDT [14359M] INFO: FTL commit: 453cdefb
2024-09-02 04:11:31.146 EDT [14359M] INFO: FTL date: 2024-08-26 21:59:41 +0200
2024-09-02 04:11:31.146 EDT [14359M] INFO: FTL user: pihole
2024-09-02 04:11:31.146 EDT [14359M] INFO: Compiled for linux/amd64 (compiled on CI) using cc (Alpine 13.2.1_git20240309) 13.2.1 20240309
2024-09-02 04:11:31.147 EDT [14359M] INFO: Wrote config file:
2024-09-02 04:11:31.147 EDT [14359M] INFO:  - 148 total entries
2024-09-02 04:11:31.147 EDT [14359M] INFO:  - 142 entries are default
2024-09-02 04:11:31.148 EDT [14359M] INFO:  - 6 entries are modified
2024-09-02 04:11:31.148 EDT [14359M] INFO:  - 0 entries are forced through environment
2024-09-02 04:11:31.148 EDT [14359M] INFO: Parsed config file /etc/pihole/pihole.toml successfully
2024-09-02 04:11:31.150 EDT [14359M] INFO: PID of FTL process: 14359
2024-09-02 04:11:31.150 EDT [14359M] INFO: listening on 0.0.0.0 port 53
2024-09-02 04:11:31.150 EDT [14359M] INFO: listening on :: port 53
2024-09-02 04:11:31.151 EDT [14359M] INFO: PID of FTL process: 14359
2024-09-02 04:11:31.151 EDT [14359M] INFO: Database version is 12
2024-09-02 04:11:31.151 EDT [14359M] INFO: Updating long-term database to version 13
2024-09-02 04:11:31.159 EDT [14359M] INFO: Updating long-term database to version 14
2024-09-02 04:11:31.162 EDT [14359M] INFO: Updating long-term database to version 15
2024-09-02 04:11:31.168 EDT [14359M] INFO: Updating long-term database to version 16
2024-09-02 04:11:31.172 EDT [14359M] INFO: Updating long-term database to version 17
2024-09-02 04:11:31.176 EDT [14359M] INFO: Updating long-term database to version 18
2024-09-02 04:11:31.181 EDT [14359M] INFO: Updating long-term database to version 19
2024-09-02 04:11:31.186 EDT [14359M] INFO: Database successfully initialized
2024-09-02 04:11:31.194 EDT [14359M] INFO: Imported 0 queries from the on-disk database (it has 0 rows)
2024-09-02 04:11:31.194 EDT [14359M] INFO: Parsing queries in database
2024-09-02 04:11:31.194 EDT [14359M] INFO: Imported 0 queries from the long-term database
2024-09-02 04:11:31.194 EDT [14359M] INFO:  -> Total DNS queries: 0
2024-09-02 04:11:31.194 EDT [14359M] INFO:  -> Cached DNS queries: 0
2024-09-02 04:11:31.194 EDT [14359M] INFO:  -> Forwarded DNS queries: 0
2024-09-02 04:11:31.194 EDT [14359M] INFO:  -> Blocked DNS queries: 0
2024-09-02 04:11:31.194 EDT [14359M] INFO:  -> Unknown DNS queries: 0
2024-09-02 04:11:31.194 EDT [14359M] INFO:  -> Unique domains: 0
2024-09-02 04:11:31.194 EDT [14359M] INFO:  -> Unique clients: 0
2024-09-02 04:11:31.194 EDT [14359M] INFO:  -> DNS cache records: 0
2024-09-02 04:11:31.194 EDT [14359M] INFO:  -> Known forward destinations: 0
2024-09-02 04:11:31.201 EDT [14359M] INFO: FTL is running as user pihole (UID 995)
2024-09-02 04:11:31.206 EDT [14359M] INFO: Created SSL/TLS certificate for pi.hole at /etc/pihole/tls.pem
2024-09-02 04:11:31.206 EDT [14359M] INFO: Reading certificate from /etc/pihole/tls.pem ...
2024-09-02 04:11:31.206 EDT [14359M] INFO: Using SSL/TLS certificate file /etc/pihole/tls.pem
2024-09-02 04:11:31.207 EDT [14359M] INFO: Restored 0 API sessions from the database
2024-09-02 04:11:31.210 EDT [14359M] INFO: Blocking status is enabled
2024-09-02 04:11:31.303 EDT [14359/T14361] INFO: Compiled 0 allow and 0 deny regex for 0 client in 0.2 msec
2024-09-02 04:11:35.449 EDT [14359/T14360] INFO: Received 8/8 valid NTP replies from pool.ntp.org
2024-09-02 04:11:35.449 EDT [14359/T14360] INFO: Time offset: -6.677707e-01 ms (excluded 2 outliers)
2024-09-02 04:11:35.449 EDT [14359/T14360] INFO: Round-trip delay: 2.585006e+01 ms (excluded 2 outliers)
2024-09-02 04:11:36.000 EDT [14359/T14360] INFO: Current RTC time is 2024-09-02 08:11:35
2024-09-02 04:11:36.000 EDT [14359/T14360] INFO: RTC time set to 2024-09-02 08:11:36
2024-09-02 04:11:36.000 EDT [14359/T14376] INFO: NTP server listening on :::123 (IPv6)
2024-09-02 04:11:36.001 EDT [14359/T14375] INFO: NTP server listening on 0.0.0.0:123 (IPv4)
```

**Comment by PromoFaux** [comment]
@yubiuser are you talking about the "Config file /etc/pihole/pihole.toml not available (r): No such file or directory" lines? I think they're pretty much expected. I put a note about it on the docker startup:

https://github.com/pi-hole/docker-pi-hole/pull/1629/files#diff-408951acb4e1dd1f37de3d4508358574cde6c06106566f250e75ab2af0cb231dR18

**Review by PromoFaux — APPROVED** [review]

**Comment by DL6ER** [comment]
@PromoFaux maybe he's talking about
```
2024-09-02 04:11:25.984 EDT [14152M] CRIT: pihole-FTL is already running!
2024-09-02 04:11:25.984 EDT [14152M] INFO: bash (PID 14155) ──> pihole-FTL (PID 14162, started 2024-09-02 04:11:25 EDT)
2024-09-02 04:11:31.145 EDT [14359M] INFO: ########## FTL started on osboxes! ##########
```

which cannot be caused by this PR but is surely not what should be happening.


TBH, I do not see anything `WARNING: ...`

**Comment by yubiuser** [comment]
I meat the SQL error on line 3. Not sure what caused it. It was a fresh install and a gravity run was performed as part of this successfully.

**Comment by DL6ER** [comment]
@yubiuser Ah, yes. That's triggered by SQlite3 and we cannot do much about it. The line following it should clarify it's noting to worry about and how to fix it:
```
[2024-09-02 04:09:30.794 13398/T13460] gravityDB_count(SELECT value FROM info WHERE property = 'gravity_count';) - SQL error step no more rows available
[2024-09-02 04:09:30.794 13398/T13460] Count of gravity domains not available. Please run pihole -g
```
That's also explaining why you see it on a fresh container.

**Pull request merged**